### ISSUE TITLE
require-hook: false warning for `test` alias

### ISSUE THREAD

**Issue by EvgenyOrekhov** [playwright-community/eslint-plugin-playwright#250]
Example [from the docs](https://playwright.dev/docs/auth#basic-shared-account-in-all-tests):

```ts
import { test as setup, expect } from '@playwright/test';

setup('authenticate', async ({ page }) => {
  // ...
});
```

Expected: no warning

Actual: "This should be done within a hook" warning

**Comment by mskelton** [comment]
I've been making some major improvements to the internals of the plugin, and this should soon be possible.
